### PR TITLE
SWTCH-1206 tests utils folder

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

File: tsconfig.spec.json [hunk 1]
```diff
@@ -12,6 +12,9 @@
     "src/test.ts",
     "src/polyfills.ts"
   ],
+  "exclude": [
+    "src/app/tests/**.ts"
+  ],
```
Comment (jrhender) [inline tsconfig.spec.json:17]: This change doesn't make alot of sense to me. I'm assuming the `tsconfig.spec.json` is the tsconfig that is used for tests, so `src/app/tests/**.ts` would be a part of that?

File: tsconfig.json [hunk 2]
```diff
@@ -27,6 +27,9 @@
     "paths": {
       "@state": [
         "src/app/state/index"
+      ],
+      "@tests": [
+        "src/app/tests/index"
```
Comment (jrhender) [inline tsconfig.json:32]: This also doesn't much sense to me. Is it because `@tests` is being imported from non test files?

**Review by jrhender — COMMENTED** [review]
Seems like some good consolidation to me, just a couple questions

File: tsconfig.json (hunk 2 above)
Comment (dawidgil) [reply tsconfig.json:32]: This is alias to file. In this case to index.ts file which exports all mocks/stubs etc. That way, i can remove paths like `../../../../../../something/.../file.ts`

File: tsconfig.spec.json (hunk 1 above)
Comment (dawidgil) [reply tsconfig.spec.json:17]: Maybe i'm wrong, but my intention was to exclude this part of code from code coverage. But after checking this, you are right, this make no difference. So i will remove this part.

File: tsconfig.json (hunk 2 above)
Comment (jrhender) [reply tsconfig.json:32]: Ah yes, right, I think you might have explained this to me before 😆

**Review by jrhender — APPROVED** [review]

**Pull request merged**